internal/services/compute/managed_disk_resource.go

```diff
@@ -713,6 +753,7 @@ func resourceManagedDiskRead(d *pluginsdk.ResourceData, meta interface{}) error
 			}
 		}
 		d.Set("trusted_launch_enabled", trustedLaunchEnabled)
+		d.Set("bursting_enabled", props.BurstingEnabled)
```

Comment:
the API doesn't necessarily return this for older resources - do we need to default this on our side:

```suggestion
		burstingEnabled := false
		if props.BurstingEnabled != nil {
			burstingEnabled = *props.BurstingEnabled
		}
		d.Set("bursting_enabled", burstingEnabled)
```

Reply:
Thanks for pointing out, updated